api-inference-community/docker_images/generic/tests/test_api_token_classification.py

```diff
@@ -11,14 +12,19 @@
     "token-classification" not in ALLOWED_TASKS,
     "token-classification not implemented",
 )
+@parameterized_class(
+    [{"model_id": model_id} for model_id in TESTABLE_MODELS["token-classification"]]
+)
```

Comment:
Not sure why you're parametrizing those tests as you're keeping only 1 model to test.
It's fine, but surprising nonetheless.

Reply:
Yes, right now we're only using one. Since I can imagine the test suite growing, specially for this use image which has many use cases, I imagined that the # of test models will probably grow over time.